diff --git a/fish_right_prompt.fish b/fish_right_prompt.fish
index 1c81e8e..ee0fbcb 100644
--- a/fish_right_prompt.fish
+++ b/fish_right_prompt.fish
@@ -52,11 +52,12 @@ function prompt_vi_mode -d 'vi mode status indicator'
 end
 
 function fish_right_prompt -d 'Prints right prompt'
-  if set -q __fish_vi_mode
-    set -l first_color black
-    set_color $first_color
-    echo "$right_segment_separator"
-    prompt_vi_mode
-    end_right_prompt
-  end
+  right_prompt_segment normal white (date +%H:%M:%S)
+  end_right_prompt
+#  if set -q __fish_vi_mode
+#    set -l first_color black
+#    set_color $first_color
+#    echo "$right_segment_separator"
+#    prompt_vi_mode
+#  end
 end
